deepform/train.py

```diff
@@ -115,14 +129,31 @@ def on_epoch_end(self, epoch, logs):
         acc_str = re.sub(r"\s+", " ", acc.to_string())
 
         print(f"This epoch {self.logname}: {acc_str}")
-        wandb.log({self.logname: acc_str})
+
+        # convert field names for benchmark logging
+        wandb.log(
+            {
+                "amount": acc_dict["gross_amount"],
+                "flight_to": acc_dict["flight_to"],
+                "flight_from": acc_dict["flight_from"],
+                "contractid": acc_dict["contract_num"],
+                "advertiser": acc_dict["advertiser"],
+            }
+        )
+        # compute average accuracy
+        wandb.log({"avg_acc": acc_dict.mean()})
```

Comment:
Oops, this also needs to change to directly use the pd.Series `acc`.

```suggestion
        wandb.log(
            {
                "amount": acc["gross_amount"],
                "flight_to": acc["flight_to"],
                "flight_from": acc["flight_from"],
                "contractid": acc["contract_num"],
                "advertiser": acc["advertiser"],
            }
        )
        # compute average accuracy
        wandb.log({"avg_acc": acc.mean()})
```

Reply:
Hmm, actually I would probably just rename and use to_dict for the log:
```suggestion
        wandb.log(acc.rename({"gross_amount": "amount", "contract_num": "contractid"}).to_dict())
        # compute average accuracy
        wandb.log({"avg_acc": acc.mean()})
```